src/styles/stop-watch.scss

```diff
@@ -0,0 +1,86 @@
+$watch-size: 500px;
+$orange-color: #e25544;
+$black-color: black;
+$minute: 60s;
+
+.container {
+  display: flex;
+  height: 100vh;
+}
+
+.stop-watch {
+  position: relative;
```

Comment:
You shouldn't set `position` and `margin` (line 14) to BEM block, only for BEM elements.
[rules](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes.html)

![Screenshot 2022-07-18 at 12 32 10](https://user-images.githubusercontent.com/17098736/179483766-5cc8efa5-5183-4824-b166-446c116b10d1.png)